sdcm/fill_db_data.py

```diff
@@ -3309,8 +3306,13 @@ def run_db_queries(self, session, default_fetch_size):
                             item["cdc_tables"][cdc_table] = self.get_cdc_log_rows(session, cdc_table)
                             LOGGER.debug(item["cdc_tables"][cdc_table])
 
+    @staticmethod
+    @retrying()
```

Comment:
I would add the exact number of retries here, so it would clearer to read.

Reply:
done